src/main/java/org/folio/consortia/controller/UserTenantController.java

```diff
@@ -0,0 +1,42 @@
+package org.folio.consortia.controller;
+
+import lombok.RequiredArgsConstructor;
+import org.folio.consortia.domain.dto.UserTenant;
+import org.folio.consortia.domain.dto.UserTenantCollection;
+import org.folio.consortia.rest.resource.UserTenantsApi;
+import org.folio.consortia.service.UserTenantService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.UUID;
+
+@RestController
+@RequestMapping("/consortia")
+@RequiredArgsConstructor
+public class UserTenantController implements UserTenantsApi {
+
+  @Autowired
+  private final UserTenantService userTenantService;
+
+  @Override
+  public ResponseEntity<UserTenantCollection> getUserTenants(UUID userId, String username, String tenantId,
+                                                             Integer offset, Integer limit) {
+    UserTenantCollection userTenantCollection;
+    if (userId != null) {
+      userTenantCollection = userTenantService.getByUserId(userId);
```

Comment:
I think we still need to support pagination for retrieving by user id, because for now we have 65 tenants and they all can be assigned to the user, so we need pagination here

Reply:
Ok, I will implement